parser_tool/templates/parser_tool/playing_with_matches_options.html

```diff
@@ -35,11 +41,24 @@
                     {{ field.label_tag }} {{ field }}
                 {% endfor %}
             </div>
-            <br>
-            <button class="btn btn-dark my-4" type="submit">Submit</button>
+            <div>Words entered: <span id="wordcount">0</span></div>
+            <button class="btn btn-primary" type="submit">Start Game</button>
+            <button class="btn btn-secondary my-4" id="generate-words">Generate Words</button>
     </form>
-       
     </div>
 </div>
+<div class="row mt-4 mb-4">
+    <div class="col-md-1"></div>
+    <div class="col-md-10">
+    <h3>Word Bank</h3>
+    <p>Select from the following list of {{ possible_words|length }} words to use in <i>Playing with Matches</i>:</p>
+    <div style="column-count: 4;">
```

Comment:
I think an improvement to this would be a searchable word bank and selected indicators for words that were generated from the quick start, but overall looks good!

Reply:
Completely agree. It would also be good to make it a little more explicit what level words are available in the word bank  (currently just level 1E words).